kitsune/search/v2/documents.py

```diff
@@ -111,47 +173,46 @@ class Index:
         using = config.DEFAULT_ES7_CONNECTION
 
     @classmethod
```

Comment:
All the `prepare_*` methods in the WikiDocument are not classmethods which probably makes sense. When we have a prepare_foo method it's going to be in most cases unique to that class. Let's remove all the classmethods for consistency.